app/mailers/patient_mailer.rb

```diff
@@ -125,7 +125,7 @@ def assessment_voice(patient)
     return unless patient.last_assessment_reminder_sent_eligible?
 
     lang = patient.select_language
-    lang = :en if %i[so].include?(lang) # Some languages are not supported via voice
+    lang = :eng if %i[som].include?(lang) # Some languages are not supported via voice
```

Comment:
Could you use the info in your new Languages.yml map to determine this now?

Reply:
I actually really like that and agree with you. Sure!

Reply:
done in 575a83062